back/config/locales/fr-BE.yml

```diff
@@ -239,3 +239,6 @@ fr-BE:
       paragraph_1_with_reason: "Votre compte sur la plateforme de participation de %{organizationName} a été temporairement désactivé pour la raison suivante :"
       for_more_information: "Pour plus d'informations à ce sujet, vous pouvez consulter le site %{termsAndConditionsPage}."
       you_can_sign_in_again_from: "Vous pouvez vous reconnecter à partir de %{date}"
+  time:
+    formats:
+      shorter: "%Hh%M"
```

Comment:
I'm guessing changes like this, other than in `en.yml` won't stick, and will instead need to be made in Crowdin.